models/user_test.go

```diff
@@ -105,3 +105,22 @@ func TestWeCanTrackTheUsersLastLogin(t *testing.T) {
 		t.Errorf("A user that has logged in should have a non-zero timestamp stored in the DB")
 	}
 }
+
```

Comment:
I think this is the first actual test I've written since... 2007? I feel so enterprise.